docs/source/tutorial/set_up_data.md

```diff
@@ -100,56 +54,95 @@ reviews:
 
- The `load_args` are passed to the `pd.read_excel` method as [keyword arguments](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_excel.html); although not specified here, `save_args` would be passed to the [`pd.DataFrame.to_excel` method](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.to_excel.html).
-```
+### Test that Kedro can load the `xlsx` data
 
-To test that everything works as expected, load the dataset within a _new_ `kedro ipython` session and display its first five rows:
+To test that everything works as expected, load the dataset within a `kedro ipython` session and display its first five rows:
 
 ```python
 shuttles = catalog.load("shuttles")
 shuttles.head()
 ```
-When you have finished, close `ipython` session as follows:
 
-```python
-exit()
+You should see output such as the following:
+
 ```
+INFO     Loading data from 'shuttles' (ExcelDataSet)
+Out[1]:
+      id       shuttle_location shuttle_type engine_type  ... d_check_complete  moon_clearance_complete     price company_id
+0  63561                   Niue      Type V5     Quantum  ...                f                        f  $1,325.0      35029
+1  36260               Anguilla      Type V5     Quantum  ...                t                        f  $1,780.0      30292
+2  57015     Russian Federation      Type V5     Quantum  ...                f                        f  $1,715.0      19032
+3  14035               Barbados      Type V5      Plasma  ...                f                        f  $4,770.0       8238
+4  10036  Sao Tome and Principe      Type V2      Plasma  ...                f                        f  $2,820.0      30342
 
-## Custom data
+```
 
-Kedro supports a number of [datasets](/kedro.extras.datasets) out of the box, but you can also add support for any proprietary data format or filesystem in your pipeline.
+When you have finished, close `ipython` session with `exit()`.
+
+## Futher information
+
+### Custom data
+
+[Kedro supports a number of datasets](/kedro.extras.datasets) out of the box, but you can also add support for any proprietary data format or filesystem in your pipeline.
 
 You can find further information about [how to add support for custom datasets](../extend_kedro/custom_datasets.md) in specific documentation covering advanced usage.
+
+### Data location
+
+Kedro uses [`fsspec`](https://filesystem-spec.readthedocs.io/en/latest/) to read data from a variety of data stores including local file systems, network file systems, cloud object stores and HDFS.
```

Comment:
The important point here is that we can work with local or ANY Cloud (AWS, Azure or GCP) this is worth emphasising over something like HDFS which is going out of fashion.